solo-chains/client/cli/src/command.rs

```diff
@@ -89,6 +90,11 @@ impl SubstrateCli for Cli {
             ]),
         )
     }
+
+    #[cfg(feature = "runtime-benchmarks")]
```

Comment:
Why do we need this in starlight but not in the other runtimes? Add a comment

Reply:
Starlight benchmarking for paras_registrar fails if we use the default config since it tries to register the parachains twice if use the standard `load_spec`